### DISCUSSION TITLE
Rigged Animation (joints?)

### DISCUSSION THREAD

**Discussion by jozkr88** [Autodesk/maya-usd#2849] [Q&A]
G'day! I wonder what type of joints/animation is this - is it procedural? The pipe moves along the path that is generated with lots of joints j

**Comment by santosg87** [comment]
hi @jozkr88 not entirely sure i understand your question here...

there are probalbly a few ways to build that, so not necessarily procedural. :)

was this maybe something for a different forum?

**Comment by jozkr88** [comment]
Thank you - could you please enlighten me what this is and how it is done? It looks like joints move the mesh along curve.

**Comment by santosg87** [comment]
To be honest, I am not sure I know the answer to either Question, based on the info you provided. your definition of what it looks like is as good as what I could tell you, as far as I can see. It indeed looks like joints and a mesh along a curve :)

is this a sample file from somewhere else? are you running into a bug? is this a USD asset? I'm not sure there is much we can provide regarding how it was made or the function of it, sorry! :)
